### PR TITLE
Location and Identity - new types, layouts and fields

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4717

## Description
Identity and Location are STIX SDOs that are being added to our XSOAR indicator types.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Identity_class.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "unknown"
+    ],
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
```
Comment (idovandijk) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-Identity_class.json:30]: Should be associated to the Identity incident type?

File: Packs/CommonTypes/IndicatorTypes/reputation-Identity.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "details": "Identity",
+    "prevDetails": "Identity",
+    "system": false,
+    "locked": false,
+    "disabled": false,
+    "file": false,
+    "updateAfter": 0,
+    "mergeContext": false,
+    "expiration": 0,
+    "layout": "b13257ad-b214-4796-8de3-8fc86c387c97",
```
Comment (idovandijk) [inline Packs/CommonTypes/IndicatorTypes/reputation-Identity.json:16]: Should be the name of the layout

File: Packs/CommonTypes/IndicatorTypes/reputation-Location.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "details": "Location",
+    "prevDetails": "Location",
+    "system": false,
+    "locked": false,
+    "disabled": false,
+    "file": false,
+    "updateAfter": 0,
+    "mergeContext": false,
+    "expiration": 0,
+    "layout": "4834667c-f521-45bb-8f9e-70643f7de336",
```
Comment (idovandijk) [inline Packs/CommonTypes/IndicatorTypes/reputation-Location.json:16]: Should be the name of the layout

**Review by idovandijk — COMMENTED** [review]
Cool

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Identity_class.json (hunk 1 above)
Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-Identity_class.json:30]: No, associated to all, we don't associate to specific types anymore.

File: Packs/CommonTypes/IndicatorTypes/reputation-Identity.json (hunk 2 above)
Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorTypes/reputation-Identity.json:16]: freaking formatters!

**Review by idovandijk — APPROVED** [review]
nice

**Pull request merged**